mobile/MyApp/App.tsx

```diff
@@ -0,0 +1,118 @@
+/**
```

Comment:
Should this file be in the PR?

Reply:
I removed it. Thanks for pointing this out :)